desktop/apps/articles/templates/section.jade

```diff
@@ -23,3 +23,6 @@ block body
         hr
         +share(section.get('title') + ' on Artsy')
   include full_feed
+
+  if veniceBanner != undefined
```

Comment:
Short answer:
The difference is so negligible that I would always pick the more readable/easier-to-understand solution.

Long answer:
It makes sense that Javascript is more performant at checking if something isn't equal to `undefined` vs. checking a truthy/falsey value. That said, _premature optimization is the root of all evil_ as [they say](https://shreevatsa.wordpress.com/2008/05/16/premature-optimization-is-the-root-of-all-evil/). The speed of JS execution is almost never a performance bottleneck unless you're doing something really intense like [parsing and compiling large graphs of stylus files on the fly](https://github.com/artsy/force/pull/1246). 95% of the performance battle with JS web apps has nothing to do with the execution speed of the language and is more about remembering not to use [synchronous APIs](https://nodejs.org/api/fs.html#fs_fs_readfilesync_file_options) within request handlers, [streaming](https://nodejs.org/api/stream.html) large data, optimizing data modeling and indexes, and keeping asset [bundle sizes small](https://github.com/artsy/force/pull/1402).